### ISSUE TITLE
examples/screenshot broken

### ISSUE THREAD

**Issue by suntong** [knq/chromedp#95]
The examples/screenshot is now broken. It ends with an endless loop looking for `#contact-form`. Please fix it. Thx. 

```
2017/07/23 16:58:05 -> {"id":360,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":361,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":361,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":362,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":362,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":363,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":363,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":364,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":364,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":365,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":365,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":366,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":366,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":367,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":367,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":368,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":368,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":369,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":369,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":370,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":370,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":371,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":371,"result":{"nodeId":0}}
2017/07/23 16:58:05 <- {"id":372,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:05 -> {"id":372,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":373,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":373,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":374,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":374,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":375,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":375,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":376,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":376,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":377,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":377,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":378,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":378,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":379,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":379,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":380,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":380,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":381,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":381,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":382,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":382,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":383,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":383,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":384,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":384,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":385,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":385,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":386,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":386,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":387,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":387,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":388,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":388,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":389,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":389,"result":{"nodeId":0}}
2017/07/23 16:58:06 <- {"id":390,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:06 -> {"id":390,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":391,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":391,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":392,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":392,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":393,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":393,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":394,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":394,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":395,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":395,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":396,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":396,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":397,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":397,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":398,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":398,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":399,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":399,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":400,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":400,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":401,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":401,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":402,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":402,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":403,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":403,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":404,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":404,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":405,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":405,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":406,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":406,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":407,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":407,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":408,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":408,"result":{"nodeId":0}}
2017/07/23 16:58:07 <- {"id":409,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:07 -> {"id":409,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":410,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":410,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":411,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":411,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":412,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":412,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":413,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":413,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":414,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":414,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":415,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":415,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":416,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":416,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":417,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":417,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":418,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":418,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":419,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":419,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":420,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":420,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":421,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":421,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":422,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":422,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":423,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":423,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":424,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":424,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":425,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":425,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":426,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":426,"result":{"nodeId":0}}
2017/07/23 16:58:08 <- {"id":427,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:08 -> {"id":427,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":428,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":428,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":429,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":429,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":430,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":430,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":431,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":431,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":432,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":432,"result":{"nodeId":0}}
2017/07/23 16:58:09 -> {"method":"DOM.attributeModified","params":{"nodeId":11,"name":"data-whatintent","value":"mouse"}}
2017/07/23 16:58:09 <- {"id":433,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":433,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":434,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":434,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":435,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":435,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":436,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":436,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":437,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":437,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":438,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":438,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":439,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":439,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":440,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":440,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":441,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":441,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":442,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":442,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":443,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":443,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":444,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":444,"result":{"nodeId":0}}
2017/07/23 16:58:09 <- {"id":445,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:09 -> {"id":445,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":446,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":446,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":447,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":447,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":448,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":448,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":449,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":449,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":450,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":450,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":451,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":451,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":452,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":452,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":453,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":453,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":454,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":454,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":455,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":455,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":456,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":456,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":457,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":457,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":458,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":458,"result":{"nodeId":0}}
2017/07/23 16:58:10 <- {"id":459,"method":"DOM.querySelector","params":{"nodeId":9,"selector":"#contact-form"}}
2017/07/23 16:58:10 -> {"id":459,"result":{"nodeId":0}}
^Csignal: interrupt
```

**Comment by adiabatic** [comment]
I got a similar error after modifying the example to point to my own site:

modified example: [main.go](https://github.com/knq/chromedp/files/1174847/main.txt)

what the console outputs: [vanity.txt](https://github.com/knq/chromedp/files/1174866/vanity.txt)

Hope this helps.

**Comment by suntong** [comment]
> I got a similar error after modifying the example to point to my own site

So you get the similar error on your own site too, right?

**Comment by kmanley** [comment]
I'm having the same issue.

**Comment by suntong** [comment]
I figured out how to take screenshots (and build a wrapper so that I can use it on command line too)

Take a look at  
https://github.com/suntong/web2image#examples

if you'd like.

**Comment by cxr002** [comment]
I am seeing this same loop issue with taking a screenshot using the simple example.
I'm on linux oel 7 with go 1.7 and google-chrome-stable-61.0.3163.100-1.x86_64 installed.
If I run google-chrome --headless --disable-gpu --screenshot https://www.google.com I get a screenshot.
I noticed if I add debugging ( --remote-debugging-port=9222) on the command line it gives an error "ERROR:headless_shell.cc(529)] Capture screenshot is disabled when remote debugging is enabled."

I have updated the main.go with the following. Having tried both runner.headless and runner.Path

      c, err := cdp.New(ctxt, cdp.WithRunnerOptions(
//          runner.Headless("/opt/google/chrome/google-chrome", 9222),
        runner.Path("/opt/google/chrome/google-chrome"),
          runner.Flag("headless", true),
          runner.Flag("disable-gpu", true),
          runner.Flag("no-first-run", true),
          runner.Flag("no-default-browser-check", true),
      ), cdp.WithLog(log.Printf))
      if err != nil {
          log.Fatal(err)
      }

Any feedback is appreciated. Thanks!
